core/src/main.cpp

```diff
@@ -267,6 +268,12 @@ constexpr auto default_jsonschema() -> std::string_view
                                         "type": "object"
                                     }},
                                     "minProperties": 1
+                                }},
+                                "user_mapping_shared_library": {{
+                                    "type": "string"
+                                }},
```

Comment:
Change `...shared_library` to:
- ~`plugin`~
- ~`path`~
- ~`full_path`~
- `plugin_path`

Should be full path.

Reply:
Done.